ksql-engine/src/main/java/io/confluent/ksql/function/udf/string/MaskKeepLeftKudf.java

```diff
@@ -49,7 +49,7 @@ public String mask(final String input, final int numChars, final String upper, f
   }
 
   private String doMask(final Masker masker, final String input, final int numChars) {
-    validateParams(numChars);
+    Masker.validateParams(this.NAME, numChars);
```

Comment:
nit: don't access static field via `this.`

Reply:
done